systems/plants/shapes/Geometry.cpp

```diff
@@ -35,6 +44,26 @@ namespace DrakeShapes
     return new Sphere(*this);
   }
 
+  void Sphere::getPoints(Matrix4Xd &points) {
+    points = Matrix4Xd::Zero(4,1);
+    points(3,0) = 1;
+  }
+
+  void Sphere::getBoundingBoxPoints(Matrix4Xd &points) {
```

Comment:
The getBoundingBoxPoints methods in Geometry subclasses have a lot of duplicated code. Should probably resolve this by creating something like a `getBoundingBoxPoints(double x_half_width, double y_half_width, double z_half_width, MatrixXd& points)` static function or protected method in Geometry.